jetty/content.md

```diff
@@ -38,6 +34,24 @@ Webapps can be [deployed](https://www.eclipse.org/jetty/documentation/current/qu
 
 For older EOL'd images based on Jetty 7 or Jetty 8, please follow the [legacy instructions](https://wiki.eclipse.org/Jetty/Howto/Deploy_Web_Applications) on the Eclipse Wiki and deploy under `/usr/local/jetty/webapps` instead of `/var/lib/jetty/webapps`.
 
+## Configuration
+
+The configuration of the Jetty server can be reported by running with the `--list-config` option:
+```console
+$ docker run -d %%REPO%%:9 --list-config
+```
+Configuration such as parameters and additional modules may also be passed in via the command line. For example:
+```console
+$ docker run -d %%REPO%%:9 --modules=jmx jetty.threadPool.maxThreads=500
+```
+To update the server configuration in a derived Docker image, the `Dockerfile` may
+enable additional modules with `RUN` commands like:
+```
```

Comment:
`markdownfmt` is still complaining about these code blocks. Each of them needs to  have a newline before and after.

Also, this particular code block should use `Dockerfile` syntax highlighting. It also doesn't need `WORKDIR` because `WORKDIR $JETTY_BASE` is already what's in effect at the end of the `Dockerfile`. Besides that, I thought that `start.jar` itself would respect the `$JETTY_BASE` environment variable and that it didn't matter which directory you ran the `--add-to-startd` command from. You'd know better than me on that point though.